### ISSUE TITLE
Per-test log files are unsupported by unix commands

### ISSUE THREAD

**Issue by electricmonk** [orbs-network/orbs-network-go#281]
After the latest change to the log files, where each test has its own file, it's hard to understand the root cause for test failures. I tried combining and sorting all files using `cat * | sort > agg.log` and got "illegal byte sequence".

Some digging around yields that macos shell commands have a hard time dealing with UTF-8-encoded text files. Are we encoding them to utf-8? if not, what's causing this?

**Comment by netoneko** [comment]
>  tried combining and sorting all files using cat * | sort > agg.log

I think you would have better time doing it with a name of actual test that fails, like this:

`cat logs/acceptance/acceptance-TestLeaderCommitsTransactionsAndSkipsInvalidOnes-* | sort`

> what's causing this

We have some binary garbage printed in the logs and I think it comes from some signature but I did not have time yet to pinpoint the location.

**Comment by electricmonk** [comment]
> I think you would have better time doing it with a name of actual test that fails, like this:

indeed, but I actually want a sorted log to figure out where the error happened

**Comment by electricmonk** [comment]
@netoneko any updates?

**Comment by netoneko** [comment]
@electricmonk do we still have the issue?
